specification/consumption/resource-manager/Microsoft.Consumption/stable/2021-05-01/examples/LotsListByBillingAccount.json

```diff
@@ -12,17 +12,32 @@
             "name": "lot1",
             "type": "Microsoft.Consumption/lots",
             "properties": {
+              "creditCurrency": "USD",
```

Comment:
Please revert all the changes in this file.

Reply:
Reverted.